content/blog/mapr-for-predictive-maintenance.md

```diff
@@ -0,0 +1,318 @@
+
+1. The first flow, located on the top half of the image below, is intended to persist IoT data and label training data for sequence prediction and anomaly detection of time-series data in Tensorflow. 
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/bi_pipeline.png" width="100%">
+
+2. The second flow, located on the bottom half, is intended to persist time-series IoT data in OpenTSDB for visualization in a Grafana dashboard. 
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/ml_pipeline.png" width="100%">
+
+Put together, these data pipelines look like this. The APIs used for reading and writing data are shown in red.
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/dataflow.png" width="100%">
+
+## Preliminary Steps
+
+These steps explain how to set up this tutorial using the [MapR Container for Developers](https://docs.datafabric.hpe.com/62/MapRContainerDevelopers/MapRContainerDevelopersOverview.html) on macOS.
+
+## Allocate 12GB to Docker
+
+This tutorial requires a lot of memory. We recommend allocating 12GB RAM, 4GB swap, and 2 CPUs to the Docker Community Edition for macOS.
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/docker_config.png" width="100%">
+
+## Start the MapR sandbox
+
+Download and run the `./mapr_devsandbox_container_setup.sh` script.
+
+```console
+git clone https://github.com/mapr-demos/mapr-db-60-getting-started
+cd mapr-db-60-getting-started
+./mapr_devsandbox_container_setup.sh
+```
+
+## Run the `init.sh` script
+
+Run the `init.sh` script to install Spark, OpenTSDB, Grafana, and some other things necessary to use the sample applications in this tutorial. SSH to the sandbox container, with password "mapr" and run the following commands. This should take about 20 minutes.
+
+```console
+ssh -p 2222 root@localhost
+wget https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/init.sh
+chmod 700 ./init.sh
+sudo ./init.sh
+```
+
+## Import the Grafana dashboard
+
+```console
+sudo /opt/mapr/server/configure.sh -R -OT `hostname -f`
+sudo /opt/mapr/opentsdb/opentsdb-2.4.0/etc/init.d/opentsdb start
+```
+
+Open Grafana data sources, with a URL like <http://maprdemo:3000/datasources/edit/1>, and add OpenTSDB as a new data source.
+
+Load the `Grafana/IoT_dashboard.json` file using Grafana's dashboard import functionality, and specify "MaprMonitoringOpenTSDB" as the data source, as shown below:
+
+<img src="https://raw.githubusercontent.com/mapr-demos/predictive-maintenance/master/images/grafana_import.png" width="100%">
+
+<hr>
+
+## Predictive Maintenance Demo Procedure
+
+For learning or debugging purposes you should run each of the following steps manually but if you just want to see data show up in Grafana then just run `./run.sh`.
```

Comment:
Add a comma just after "...the following steps manually"

Reply:
done